build-release.sh

```diff
@@ -10,7 +10,8 @@ export OUT_DIR="${1-dist}"
 export VERSION=${2:-$(git describe --tags --always --dirty)}
 
 # To overwrite the version details, pass something as the third arg. Empty string disables it.
-export VERSION_DETAILS=${3-"$(date -u +"%FT%T%z")/$(git describe --tags --always --long --dirty)"}
+export COMMIT_DATE=$(git log -1 --format=%at $VERSION | { read d ; date -u +"%FT%T%z" -d @"$d" ; })
```

Comment:
How about using `--format=%aI`, it could help us have no conversion magic?

```bash
git log -1 --format=%aI 48dddc3005c9fbc2fd9a52990560790535a29f69
2023-09-05T15:27:34+02:00
```

Reply:
I tried but it uses local time, so in this way, we would have a dependency on the enviroment's timezone. And, I didn't find a way to use `git log` for dumping an ISO8601 date time in UTC.

Reply:
I see. Then yeah makes sense :+1: 

And the last one juts double-checking if we want to use the author datetime instead of the commuter datetime https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/troubleshooting-commits-on-your-timeline#how-github-uses-the-git-author-date-and-commit-date ?

Reply:
Actually :arrow_up: make nos sense since our main use case where version is the tag :thinking:

Reply:
```suggestion
export COMMIT_DATE=$(git log -1 --format=%ct $VERSION | { read d ; date -u +"%FT%T%z" -d @"$d" ; })
```
Instead, I think you're right it matters. If GitHub builds in one way and on the local build we print in another way then we are again environment-dependent. Let's force to use the COMMIT_DATE all the time.